packages/tpetra/core/src/Tpetra_BlockCrsMatrix_def.hpp

```diff
@@ -366,6 +366,7 @@ namespace Impl {
     KOKKOS_INLINE_FUNCTION void
     operator () (const typename Kokkos::TeamPolicy<typename device_type::execution_space>::member_type & member) const
     {
+      constexpr bool runOnHost = !std::is_same_v<typename Kokkos::TeamPolicy<typename device_type::execution_space>::member_type, Kokkos::DefaultExecutionSpace> || std::is_same_v<Kokkos::DefaultExecutionSpace, Kokkos::DefaultHostExecutionSpace>;
```

Comment:
this doesn't make sense??

Reply:
is this meant to be:
```c++
constexpr bool runOnHost = !std::is_same_v<typename device_type::execution_space, Kokkos::DefaultExecutionSpace> || std::is_same_v<Kokkos::DefaultExecutionSpace, Kokkos::DefaultHostExecutionSpace>
```

Reply:
I didn't know github can do code suggestions like how @dalg24 has been doing.  Very cool!